### DISCUSSION TITLE
Mingda Magician printers

### DISCUSSION THREAD

**Discussion by Matrekus** [OrcaSlicer/OrcaSlicer#2035] [General]
Hello, I made the profiles for Mingda Magician X, Max, Pro.
Took the profiles according quite similar printers, but they are not tested and have maybe some things to do.

Is someone interested to test and finalize them. I have only the pro, so I cant test the other ones.

Maybe there is a chance to add them in the printer selection when finished. 

![Mingdas](https://github.com/SoftFever/OrcaSlicer/assets/100630918/947db7a5-acb3-479f-a7db-410bd302111a)

**Comment by ff2emtvol** [comment]
I have a Magician Max and would be happy to test. What printer did you first use and what changes did you make. I do not get past the select printer install page because there are no Mingda printers listed.

**Comment by Georgelou35** [comment]
Nice i woold love to try these settings..I have magician max.And i hate cura.

**Comment by Georgelou35** [comment]
Where i go to download it???

**Comment by cdponcedeleon** [comment]
I have a Magician X 1st gen and would love to give it a whirl and go back with comments/adjustments necessary.

**Comment by RadsWa** [comment]
If this was not already done, I'd be happy to test for the magician X

**Comment by UglyCodeGuy** [comment]
Hi Matrekus. I have the Mingda Magician X. Im interested to test your profile. Are you willing to send it over?

**Comment by Steffen2024** [comment]
Hello. I am interested. I have an Magican Pro 2 :)

**Comment by Georgelou35** [comment]
Please can you send them to my email to test them..
Con855@hotmail.com.
thank you very much

**Comment by phoenixtomb** [comment]
I also have a Magician X, and would like o test it the profile if possible.

**Comment by Georgelou35** [comment]
Guys he doesnt reply...i dont know why.

**Comment by sattaway88** [comment]
Can I get those profiles please?

**Comment by FSMaker77** [comment]
slicer exist, fork of orcaslicer https://www.3dmingda.com/software

**Reply by rlmarin** [reply]
Their Orca fork doesnt have profiles for the older models.

**Comment by gljony** [comment]
Tengo una mingda magician pro estaria interesado en los perfiles

**Comment by LuckyPuccio89** [comment]
i published the profiles for the newest x2 models too
if you want here https://github.com/LuckyPuccio89/MINGDA_ORCASLICER.git

**Comment by pielorz** [comment]
@Matrekus Did you finish the Magician X gen 1 profiles? If so, can we have a link to download as they are not in the Orca software.

**Reply by dssoftx** [reply]
I have 2 tuned profiles here. 
https://github.com/DsSoft-Byte/MagicianProLegacy
R Profile is Noise and Reliability optimised. (1H57min Benchy)
S Profile is Pure Speed with reduced quality (48min Benchy)

**Reply by pielorz** [reply]
I really appreciate this, cant wait to give it a go. Thanks @DsSoft-Byte

**Reply by dssoftx** [reply]
> I really appreciate this, cant wait to give it a go. Thanks @DsSoft-Byte

The Speed profile has a bug. I already have a fixed profile and i just need to upload it. Belt skipping on this printer is a nightmare.

**Comment by dssoftx** [comment]
https://github.com/DsSoft-Byte/MagicianProLegacy
